### ISSUE TITLE
Make probot.glitch.me an official Probot project

### ISSUE THREAD

**Issue by gr2m** [probot/ideas#31]
https://github.com/gr2m/glitch-github-app is the code behind https://probot.glitch.me, a web form to create a new Probot App and connect it to a new GitHub app with instructions.

I use it quite a lot myself to quickly "fork" existing probot aps and play around with their code, but there are a few things that could be improved. For example

- call the Glitch Remix URL in an iframe (if possible), so that the user ends up directly at the form instead of at the Glitch Editor from where they have to press the ![glitch show](https://user-images.githubusercontent.com/39992/36068560-1d5cf894-0e8d-11e8-9445-dc0ec604e62f.png) button.
- Instead of providing just the textfield to enter a repository name, it could be prefilled with the apps from https://probot.github.io/apps/. Setting a custom repository would also be an option. 
- It would be nice if I could preset parameters, e.g. for the URL that I want the bot to be based on, but also the permissions. I think the https://github.com/settings/apps/new URL accepts query arguments so we could provide a link to create a new app that has the fields and permissions/event hooks already preset
- There currently is a bug that the causes the app's dependencies not to be installed correctly. I need to open the terminal and `rm -rf node_modules` and then npm install again. That was not necessary before, I'm sure we can find out to fix that

That would be a great project for Google / Rails Girls Summer of Code team, too, I think. 

Any thoughts on moving https://github.com/gr2m/glitch-github-app to the @probot organization and making it officially a community project? @probot/maintainers

**Comment by tcbyrd** [comment]
I'm all 👍 for making this an official project. Just heads up that I tried using URL parameters on the new app form and couldn't make it work in the same way that it works on new issues/PRs, so I'm not sure if that's possible today. Maybe something to raise as ecosystem feedback.

@gr2m Just curious if you've also looked into the Glitch [for Platforms](https://glitch.com/forplatforms/) and if that has the capabilities we'd need to support importing an example app from a form on `probot.github.io`? I wonder if it could create the unique Glitch URL after you fill out the form instead of the form living on a Glitch and rewrite the filesystem. It looks like you can set environment variables from URL parameters in this generator: https://remix-button.glitch.me/

**Comment by gr2m** [comment]
> Just heads up that I tried using URL parameters on the new app form and couldn't make it work in the same way that it works on new issues/PRs

damnit, I think you are right. It works with tokens: https://github.com/settings/tokens/new?description=github-rename-tags-and-releases&scopes=read:org,user:email. I've left a message i the `#github-feedback` slack channel about it.

> if you've also looked into the Glitch for Platforms

Yes! I wanted to make a case first and then ask if GitHub would sponsor that :) I think the integration would be much nicer. If we made it official, it would be nice to pay Glitch for the amazing value they create for the Probot community, too.

> It looks like you can set environment variables from URL parameters 

Yes you can pass environment variables to Glitch remix URLs :)

**Comment by gr2m** [comment]
Update: I need to update the way probot.glitch.me works before we can move on, so that’s on hold until then
